{
  assertEquals(true,result.isFailed());
  LOG.info(result.getExceptionFullMessage());
  Throwable cause=getExceptionCause(result);
  assertTrue("expected abort exception, got " + cause,cause instanceof ProcedureAbortedException);
}
